src/components/dashboard/employees/Employees.js

```diff
@@ -36,40 +39,45 @@ class Employees extends Component {
   };
 
   render() {
+    const { employees } = this.props;
+
     return (
       <div className="employees">
         <div className="employees-container">
           <TitleBar
             title="Manage your employees"
             subtitle="Add a new employee to send them a new hire box"
           />
-          {this.props.employees.length <= 0 && (
-            <div className="employees-zero-state">
-              <div className="employees-zs-text">
-                You haven't added any employees to send swag to yet! Start adding employees by clicking
-                on the add button.
-              </div>
-              <div className="contain-zero-state-actions">
-                <span className="create-button" onClick={this.onCreateClick}>
-                  <MdAddCircle />
-                </span>
-              </div>
-            </div>
-          )}
-          {this.props.employees.length > 0 && (
-            <div>
-              <div className="contain-actions">
-                <span className="create-button" onClick={this.onCreateClick}>
-                  <span className="create-employee-text">Add Employee</span>
-                  <MdAddCircle />
-                </span>
+          {employees !== null &&
+            employees.length <= 0 && (
+              <div className="employees-zero-state">
+                <div className="employees-zs-text">
+                  {`You haven\'t added any employees to send swag to yet! Start adding employees by
+                  clicking on the add button.`}
+                </div>
+                <div className="contain-zero-state-actions">
+                  <span className="create-button" onClick={this.onCreateClick}>
+                    <MdAddCircle />
+                  </span>
+                </div>
               </div>
-              <div className="employees-table">
-                <Table data={this.props.employees} columns={columns} />
+            )}
+          {employees !== null &&
+            employees.length > 0 && (
+              <div>
+                <div className="contain-actions">
+                  <span className="create-button" onClick={this.onCreateClick}>
+                    <span className="create-employee-text">Add Employee</span>
+                    <MdAddCircle />
+                  </span>
+                </div>
+                <div className="employees-table">
+                  <Table data={this.props.employees} columns={columns} />
+                </div>
+                <div className="total-row">Total Employees: {this.props.employees.length}</div>
               </div>
-              <div className="total-row">Total Employees: {this.props.employees.length}</div>
-            </div>
-          )}
+            )}
+          {employees === null && <Loader />}
```

Comment:
Could this just be `this.state.loading && <Loader />`  to be a little easier to read?